pkg/controller/cronjob/utils_test.go

```diff
@@ -25,7 +25,7 @@ import (
 
 	cron "github.com/robfig/cron/v3"
 	batchv1 "k8s.io/api/batch/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
You don't need to add an alias here.

Reply:
i think its done by vscode plugin automatically, i can remove it if needed

Reply:
Yes please.

Reply:
updated.